src/app/Library/CrudPanel/Traits/WarnBeforeLeaving.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\CrudPanel\Traits;
+
+trait WarnBeforeLeaving
```

Comment:
@pxpm what do you think about this `Trait`?
Is it justifiable? Or is it too much detail?

Reply:
I made the trait based on [his suggestion](https://github.com/Laravel-Backpack/CRUD/pull/3616#discussion_r620360895).

Reply:
Ah sure! never mind 😅